### PR TITLE
Code refactor and clean up

### PR DESCRIPTION
**Pull request by kiransuren**
This PR refactors and cleans up the repository.

### CODE REVIEW

**Review by concon500 — APPROVED** [review]
This seems okay.

File: Core/Src/main.c [hunk 1]
```diff
@@ -45,17 +45,17 @@ TIM_HandleTypeDef htim1;
...
 UART_HandleTypeDef huart2;
 
-/* Definitions for defaultTask */
-osThreadId_t defaultTaskHandle;
-const osThreadAttr_t defaultTask_attributes = {
-  .name = "defaultTask",
+/* Definitions for heartbeatTsk */
+osThreadId_t heartbeatTskHandle;
+const osThreadAttr_t heartbeatTsk_attributes = {
+  .name = "heartbeatTsk",
```
Comment (Blaine-Martian) [inline Core/Src/main.c:51]: please change this name

File: Core/Src/main.c [hunk 2]
```diff
@@ -351,53 +351,50 @@ static void MX_GPIO_Init(void)
...
+  /* USER CODE BEGIN grabberMotorTaskFunc */
+  /* This task moves the grabber servo motor from 0degrees to 180degrees then resets to 0degrees and repeats*/
+  // 50 -> 100 : 0deg to 180deg linearly
+  int i = 50;
   for(;;)
   {
-	if(i==25) break;
+	if (i==100) {
+		i=50;
+	}
```
Comment (Blaine-Martian) [inline Core/Src/main.c:392]: Please explain this

**Review by Blaine-Martian — CHANGES_REQUESTED** [review]
Overall it was okay, please change small comments.

File: Core/Src/main.c (hunk 1 above)
Comment (kiransuren) [reply Core/Src/main.c:51]: no this not problem

**Review by concon500 — APPROVED** [review]
Once again, this is good, Dawson is lying.

**Review by Blaine-Martian — APPROVED** [review]
LGTM

**Pull request merged**